src/main/java/pl/sda/javapoz/model/FilterProducts.java

```diff
@@ -1,20 +1,14 @@
 package pl.sda.javapoz.model;
 
+import javax.persistence.Entity;
 import java.util.Date;
 
-/**
- * Created by RENT on 2017-03-22.
- */
-
+@Entity
```

Comment:
To raczej tez nie jest encja tylko, przefiltrowane produkty.

Reply:
Wyjasnione